example/addons/tiltfive/T5ProjectSettings.gd

```diff
@@ -0,0 +1,48 @@
+class_name T5ProjectSettings extends RefCounted
```

Comment:
This gave an issue for me when it wasn't a tool script, because it was used from another tool script

Reply:
Also since all access is static, not sure if this needs to extend `RefCounted`, it can just extend `Object`?

Reply:
I added @tool and changed it to object. I also put the godot-cpp submodule at version 4.1